boilerplate/update

```diff
@@ -140,6 +140,9 @@ if [ ! -f "$CONFIG_FILE" ]; then
   exit 1
 fi
 
+# The most recent build image tag. Export this for individual `update` scripts.
+export LATEST_IMAGE_TAG=$(cd $BP_CLONE; git describe --tags --abbrev=0 --match image-v*)
```

Comment:
Shouldn't there be a ` | head -n1` to select only the latest tag?

Reply:
Per the [man page](https://git-scm.com/docs/git-describe), `git describe` already only finds the most recent tag. (I just sanity-checked this locally -- it's true :) )